workspaces/azure-storage-explorer/plugins/azure-storage-backend/README.md

```diff
@@ -0,0 +1,98 @@
+# Azure Storage Explorer Backend
+
+Welcome to the `Azure Storage backend` plugin!
+
+## Getting started
+
+### Installation
```

Comment:
We should have instructions for installing this using the new backend system and then this should be called legacy backend.